pkg/directory/farms_handlers.go

```diff
@@ -19,6 +21,10 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
```

Comment:
This is not the right way to do this because this can cause conflicts instead what u need to do is 
```go
type farmKey struct{}

// then to set up the context
ctx = context.WithValue(ctx, farmKey{}, <value>)

// and get it like
v := ctx.Value(farmKey{})
```

But to make it even cleaner usually a middleware defines another method like `GetFarmID` like

```go
func GetFarmID(ctx context.Context) schema.ID {
   return ctx.Value(farmKey{}).(schema.ID)
}
```
So the user of the ctx does not care about the key type. and having a struct as a key type make sure no conflicts can happen